lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,81 @@
+= Exploring the Fuel Efficiency Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to the  @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `mpg-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
```

Comment:
recommend moving these notes on the columns to comments in the starter file and pointing out their existence there to students.

Reply:
done